src/include/mallocMC/creationPolicies/Scatter_impl.hpp

```diff
@@ -182,7 +179,7 @@ namespace ScatterKernelDetail{
        * The page struct is used to access the data on the page more efficiently
        * and to clear the area on the page, which might hold bitsfields later one
        */
-      struct PAGE
+      struct PAGE // TODO(bgruber): why is this in ALLCAPS?
```

Comment:
I think it was Caps in the original scatter implementation and is copy pasted.

I would suggest to change it to `Page` to fit our [coding style](https://github.com/ComputationalRadiationPhysics/contributing/blob/master/codingGuideLines/cpp.md), note: The rest of the code also do not fit the coding style because the code is older than our style guide.

Reply:
About the coding style: I guess nobody had time yet to put together a .clang-format file? That would enable automatic bulk formatting from the command line and some IDEs (VS for sure, CLion probably as well) can automatically pick up the file if it exists and format while typing.

Reply:
That's a great question! We had alpaka effort for it [here](https://github.com/alpaka-group/alpaka/pull/781). The issue is, we can't make it support template formatting like we want to have it, and a few of our projects heavily employ templates.

Reply:
@bernhardmgruber It would be nice to have clang format but we have heavily templated code in many project therefore the code style is a little bit special in the template formation part. Clang format has no useful rules to handle templates. For functions/methods you can place each argument on a seperate line and you can define how parentheses will be handled (east vs. west coast style). Template parentheses `<`, `>` will not be handled as parentheses :-(

Reply:
I renamed PAGE to Page.

Hmm, what was the last version of clang-format that you have tried? There are constant improvements.
And thank you for the styleguide instructions! But I work in too many projects (also hobby projects), which all have different coding styles. So clang-format files in every repository work superbly here! So having one would already help a lot, even if it's not perfect! I will have a look at the file for alpaka.

Reply:
After a talk with @psychocoderHPC today I will try to get a .clang-format file working at some point. But I think it does not need to be part of this PR.